src/gui/mainwindow.h

```diff
@@ -103,7 +103,7 @@ class MainWindow: public QMainWindow
     void showNotificationBaloon(QString title, QString msg) const;
 
 private slots:
-    void toggleVisibility(QSystemTrayIcon::ActivationReason e = QSystemTrayIcon::Trigger);
+    void toggleVisibility(QSystemTrayIcon::ActivationReason reason = QSystemTrayIcon::Trigger);
```

Comment:
> Don't pass parameters of POD types as const reference, since it doesn't have any sense.

Why? Consider
```c++
struct S {
    double ar[100];
}
```